scripts/e2e-using-tekton/e2e-pipeline.yaml

```diff
@@ -0,0 +1,100 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pipeline-sa-admin
+subjects:
+  - kind: ServiceAccount
+    name: pipeline
+    namespace: gitpos-e2e-test
+roleRef:
+  # Operator-sdk tries to create cluster-scoped resources.
+  # Doesn't hurt to make the test pipeline sa a cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+---
+kind: Task
+metadata:
+  name: run-operator-sdk-e2e
+  namespace: gitpos-e2e-test
+spec:
+  workspaces:
+  - name: source
+  - name: tmp 
+    mountPath: /tmp/output
+  steps:
+    # uses the specific version of operator-sdk that our tests need
+    # to run on. We may replace this with a better maintained image.
+    # Built using Dockerfile.operator-sdk in this source-code-repo
+  - image: quay.io/shbose/operator-sdk
+    workingDir: $(workspaces.source.path) 
+    env:
+      - name: XDG_CACHE_HOME
+        value: /tmp/output/.cache
+    command:
+      - make
+      - test-e2e-on-installed-operator
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: gitops-osdk-e2e
+  namespace: gitpos-e2e-test
+spec:
+  params:
+  - name: repo-url
+    type: string
+    description: The git repository URL to clone from.
+  - name: branch-name
+    type: string
+    description: The git branch to clone.
+  workspaces:
+  - name: shared-data # needed to check out source code
+  - name: tmp # needed to write out a kubeconfig for operator-sdk to consume.
+  tasks:
+  - name: fetch-repo
+    taskRef:
+      name: git-clone
+      kind: ClusterTask 
+    workspaces:
+    - name: output
+      workspace: shared-data
+    params:
+    - name: url
+      value: $(params.repo-url)
+    - name: revision
+      value: $(params.branch-name)
+  - name: run-e2e
+    runAfter: ["fetch-repo"]  # Wait until the clone is done before reading the readme.
+    workspaces:
+    - name: source
+      workspace: shared-data
+    - name: tmp 
+      workspace: tmp
+    taskRef:
+      name: run-operator-sdk-e2e
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  namespace: gitpos-e2e-test
+  generateName: gitops-osdk-e2e-
+spec:
+  pipelineRef:
+    name: gitops-osdk-e2e
+  workspaces:
+  - name: tmp 
+    emptyDir: {}
+  - name: shared-data
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+  params:
+  - name: repo-url
+    value: https://github.com/redhat-developer/gitops-operator
```

Comment:
won't work since current master doesn't have the changes - use https://github.com/sbose78/gitops-operator / branch: dockerfile.e2e`